app/assets/stylesheets/style.css

```diff
@@ -109,11 +109,11 @@ a.progress_link:hover {
 
 #main {
   top: 30px;
-  left: 200px;
+  margin-left: 200px;
   right: 0;
   bottom: 0;
-  position: fixed;
   overflow-y: auto;
+  outline: none;
```

Comment:
What is the function of adding `outline: none`?

Reply:
Hovering or clicking inside the main container (problem statement area) creates a "selected" outline.

Reply:
The default behaviour of outline should be none by default. This seems to be a consequence of the `tabindex="-1"` attribute. The only two reasons I can think of for needing `tabindex="-1"` is for accessibility and to facilitate automated tests. As far as accessibility, removing it doesn't seem to change the tab order of the page. If coveralls returns the same result, I would be inclined to remove the tabindex attribute instead

Reply:
Per mirrored commit on https://github.com/NZOI/nztrain/pull/80, coveralls remains unchanged.